src/components/AccountDetailsV2/index.tsx

```diff
@@ -1,6 +1,6 @@
+import { SupportedChainId } from '@uniswap/sdk-core'
```

Comment:
hmm i think i would still prefer us to define our own `SupportedChainId` on the interface rather than strictly taking from sdk-core

i can see a scenario in which sdk-core starts supporting a chain id that we don't - for example, adding a testnet, and then that breaks some of our typescript handling 

but not too picky about it

Reply:
for example, i think in this scenario you wouldn't have had to add arbitrum goerli to also meet ts lint checking, but not a big deal

Reply:
I'm also curious abt why we're switching away from defining our own SupportedChainId, as it changes the meaning of something like `isSupportedChain` from _interface supports this chain_ => _the sdk supports this chain_

Reply:
I think about it as *Uniswap Labs* supports X chain, which is why I think a canonical sdk chain definition makes sense.

Reply:
That may not be the case across all platforms though. Sdk-core is meant to be shared across apps and [redacted] may have different chain support than interface in the future.